omod/src/main/java/org/openmrs/module/appointments/web/contract/AppointmentsSummary.java

```diff
@@ -30,4 +50,29 @@ public Map<String, DailyAppointmentServiceSummary> getAppointmentCountMap() {
     public void setAppointmentCountMap(Map<String, DailyAppointmentServiceSummary> appointmentCountMap) {
         this.appointmentCountMap = appointmentCountMap;
     }
+
+    public List<DailyAppointmentServiceSummary> getServices() {
+        return services;
+    }
+
+    public void setServices(List<DailyAppointmentServiceSummary> dailyAppointmentList) {
+        this.services = dailyAppointmentList;
+    }
+
+    public String getAppointmentDate() {
+        return appointmentDate;
+    }
+
+    public void setAppointmentDate(String appointmentDate) {
+        this.appointmentDate = appointmentDate;
+    }
+
+    public Integer gettotalServiceCount() {
```

Comment:
This applies to the setter as well:

```suggestion
    public Integer getTotalServiceCount() {
```